app/src/main/kotlin/de/digitalService/useID/ui/screens/identification/IdentificationAttributeConsent.kt

```diff
@@ -50,42 +48,43 @@ fun IdentificationAttributeConsent(
         navigationButton = NavigationButton(
             icon = if (viewModel.didSetup) NavigationIcon.Back else NavigationIcon.Cancel,
             shouldShowConfirmDialog = !viewModel.didSetup,
-            onClick = viewModel::onCancelButtonTapped,
+            onClick = viewModel::onCancelButtonClicked,
             isIdentification = true
         )
     ) { topPadding ->
         Scaffold(bottomBar = {
-            RegularBundButton(
+            BundButton(
                 type = ButtonType.PRIMARY,
-                onClick = viewModel::onPINButtonTapped,
+                onClick = viewModel::onPinButtonClicked,
                 label = stringResource(id = R.string.identification_attributeConsent_continue),
                 modifier = Modifier
-                    .padding(20.dp)
+                    .padding(UseIdTheme.spaces.m)
             )
         }, modifier = modifier.padding(top = topPadding)) { paddingValues ->
             Column(
                 modifier = Modifier
                     .padding(bottom = paddingValues.calculateBottomPadding())
-                    .padding(horizontal = 20.dp)
+                    .padding(horizontal = UseIdTheme.spaces.m)
                     .verticalScroll(rememberScrollState())
             ) {
                 Text(
                     viewModel.identificationProvider,
-                    style = MaterialTheme.typography.titleLarge
+                    style = UseIdTheme.typography.headingXl
                 )
-                Spacer(modifier = Modifier.padding(5.dp))
+                Spacer(modifier = Modifier.height(UseIdTheme.spaces.m))
                 Text(
                     stringResource(
                         id = R.string.identification_attributeConsent_body,
                         viewModel.identificationProvider
                     ),
-                    style = MaterialTheme.typography.bodySmall
+                    style = UseIdTheme.typography.bodyLRegular
                 )
+                Spacer(modifier = Modifier.height(UseIdTheme.spaces.m))
                 AttributeList(attributeIDs = viewModel.requiredReadAttributes)
-                Spacer(Modifier.height(20.dp))
+                Spacer(modifier = Modifier.height(UseIdTheme.spaces.m))
                 BundButton(
```

Comment:
The layout of this button seems to be broken. (I think this might have been introduced by my refactoring of the Bund Button. 😇)

Reply:
I am not sure what you mean, this is what it looks like in my Simulator:
![image](https://user-images.githubusercontent.com/33418462/207286974-efcefe27-136c-4798-b577-4d7b38277612.png)